src/langs/fr.json

```
@@ -79,6 +79,8 @@
   "BOOKMARKED_ITEMS_TITLE": "Signets",
   "FLAG_ITEM_BUTTON": "Signaler",
   "FLAG_ITEM_MODAL_TITLE": "Signaler l'élément",
+  "HIDE_ITEM_SETTING_TITLE": "Cacher pour les lecteurs",
+  "HIDE_ITEM_SETTING_DESCRIPTION": "Si cette option est activée, les lecteurs ne peuvent pas accéder à cet élément ni à son contenu.",
   "HIDE_ITEM_HIDDEN_PARENT_INFORMATION": "Cet élément est caché car son élément parent est caché.",
```

If you prefer, reuse "caché" twice:
```suggestion
  "HIDE_ITEM_HIDDEN_PARENT_INFORMATION": "Cet élément est caché, car son parent est également masqué.",
```